### ISSUE TITLE
Self test fails for YOLOv5Net

### ISSUE THREAD

**Issue by ChrisDodgeSymetri** [codeproject/CodeProject.AI-Server#166]
Windows version 2.9.2, happens on 2 machines...   I dont know if it happened with the previous version.

I noticed during initialize the log mentioned `Self test: Self-test failed` for YOLOv5Net.

When I run the selftest manually it also fails:

```
C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\bin>ObjectDetectionYOLOv5Net.exe --selftest
fail: CodeProject.AI.Modules.ObjectDetection.YOLOv5.ObjectDetector[0]
      Unable to load the model at C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\bin\assets\yolov5m.onnx
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
      Content root path: C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\bin
info: Microsoft.Hosting.Lifetime[0]
      Application is shutting down...

```

Temp Fix:   Copy:

C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\assets

To

C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\bin\assets

Now self test passes.

WOULD that have caused any deeper issues?  

And maybe this folder needs moving also?
C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\custom-models

This may be the source of the bug?

C:\Program Files\CodeProject\AI\modules\ObjectDetectionYOLOv5Net\install.bat
`call "%utilsScript%" GetFromServer "binaries/" "!imageName!" "bin" "Downloading !imageName!..."
`

**Comment by ChrisDodgeSymetri** [comment]
2.9.3 installer, self test still fails during install for YOLOv5Net and the module service does not start because of that.   It's still installed, just not started.   Yolov6 6.2 installs and runs fine.

I did a CLEAN install (deleted ProgramData and Program Files) CodeProject folder.   <----  This may be why not everyone can replicate?

**Comment by ChrisMaunder** [comment]
Can you please try re-downloading and installing 2.9.3 again? Should be good now.

**Comment by ChrisDodgeSymetri** [comment]
Self test error fixed!  However, the .net module does not auto start even though it passed and I had it checked during the install. 

But once i hit start it stays going even after re-starting the service.   Seems to work fine.      This was a clean test, I removed both CodeProject folders before re-installing so that could be a factor for replicating.

I tested on 2 machines, I had to start .net manually on both.
